### DISCUSSION TITLE
WebView2.EnsureCoreWebView2Async() throws a COMException on Uno.Sdk 5.5.49

### DISCUSSION THREAD

**Discussion by trydalch** [unoplatform/uno#18988] [Q&A]
After upgrading from Uno.Sdk 5.3.99 to 5.5.49, I'm getting a runtime exception with the WebView2 when running on Windows. I get no such exception on iOS. 

The exception is thrown on the `EnsureCoreWEbView2Async()` method call. 

![image](https://github.com/user-attachments/assets/fdba03c3-99fe-4902-bbce-cde5788f98ca)

This wasn't happening on earlier versions.

**Comment by trydalch** [comment]
The issue does not present on 5.3.112.

**Reply by jeromelaban** [reply]
See https://github.com/unoplatform/uno/issues/18552#issuecomment-2489673680 for a workaround.

**Reply by trydalch** [reply]
Thanks @jeromelaban
